docs/how-to-guides/themes/theme-json.md

```diff
@@ -1084,6 +1085,41 @@ Pseudo selectors `:hover`, `:focus`, `:visited`, `:active`, `:link`, `:any-link`
 	}
 ```
 
+#### Variations
+
+A block can have a "style variation", as defined per the [block.json specification](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-registration/#styles-optional). Theme authors can define the style attributes for an existing style variation using the `theme.json`. Styles for unregistered style variations will be ignored.
+
```

Comment:
What I mean is, if I register a new custom block style in my theme, it is in fact _registered_, but as I understood it, styling it doesn't work since it is not a core block style variation.

Reply:
I got confused, it is the **UI** only, that does not work for variations that are not added by core.

Reply:
So, what do you think of the new section? Anything missing or do you ship it? :)